app/Http/Controllers/ReportController.php

```diff
@@ -8,23 +8,23 @@
 
 class ReportController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->